content/post/contributors/yokwejuste.md

```diff
@@ -0,0 +1,10 @@
+---
+title: "Yokwejuste"
+date: 2022-01-19T14:53:19+01:00
+images: []
+categories: ["Web development", "Data Science", "Blockchain", "Cloud Computing"]
+tags: ["Open-source", "github"]
+authors: ["yokwejuste"]
+---
+Hi, I'm Yokwejuste, a young inspired blogger, aspiring to be a computer engineering. I like trying everything and being involved in everything tech 😅.
+The fields in which I'm involved include: Data Science, Web Development, blockchain, Cloud Computing and AI.
```

Comment:
```suggestion
The fields in which I'm involved are: Data Science, Web Development, blockchain, Cloud Computing and AI.
```

Reply:
That's true, grammatical error, but still the same sense I guess!